### PR TITLE
Add chrome,tray and webview relative api in nwjs project.

### PR DESCRIPTION
**Pull request by xiaoyu3x**
chrome api:chrome.contentSettings/tabs/proxy/cookies
tray api:tray.remove()/event:click
webview:webview.showDevTools(show, [container])

| | Pass | Fail | Block|
---|---|---|---
Total |  13 | 0 | 0
Auto|  0 | 0 | 0
Manual | 13 | 0| 0

### CODE REVIEW

File: nwjs-iot/nwjsapi/chrome-extension-api/chrome-cookies/index.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  })
+}
+var details = {name: "testcookie", url: "http://www.baidu.com"};
+function getCookie() {
+  chrome.cookies.get(details, function(cookie) {
+    document.getElementById("cookie").innerHTML = cookie.value;
+  })
+}
+chrome.cookies.onChanged.addListener(function() {
+  document.getElementById("demo").innerHTML = "Set Cookie Success";
```
Comment (Honry) [inline nwjs-iot/nwjsapi/chrome-extension-api/chrome-cookies/index.html]: See the text "Set Cookie Success" should be part of expected result.

File: nwjs-iot/nwjsapi/chrome-extension-api/chrome-proxy/index.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="author" title="zhang xiaoyux" href="mailto:xiaoyux.zhang@intel.com">
+<link rel="help" href="https://developer.chrome.com/extensions/proxy">
+<p>
+  <strong>Test Steps:</strong>
+</p>
+<ol>
+  <li>Click "Set Proxy" button to set proxy.</li>
+  <li>Click "Get Proxy" button to get proxy.</li>
+  <li>Click "Clear Proxy", then click "Get Proxy".</li>
```
Comment (Honry) [inline nwjs-iot/nwjsapi/chrome-extension-api/chrome-proxy/index.html]: Why not to checking if the proxy is set successfully? You just check the proxy can be set and got.

File: nwjs-iot/nwjsapi/chrome-extension-api/chrome-runtime-getPlatformInfo/index.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  <strong>Test Steps:</strong>
+</p>
+<ol>
+  <li>Click "Show Platform Info" to show platform information.</li>
+</ol>
+<p>
+  <strong>Expected Output:</strong>
+</p>
+<ol>
+  <li>"os is: linux" and "arch is: x86-64" are showed if platform is linux x86-64.</li>
```
Comment (Honry) [inline nwjs-iot/nwjsapi/chrome-extension-api/chrome-runtime-getPlatformInfo/index.html]: Just tell people that the platform info is shown.

File: nwjs-iot/nwjsapi/chrome-extension-api/chrome-tabs/index.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  <strong>Test Steps:</strong>
+</p>
+<ol>
+  <li>Click "Create Tabs" to open Baidu in the browser.</li>
+</ol>
+<p>
+  <strong>Expected Output:</strong>
+</p>
+<ol>
+  <li>Baidu is opened and "success" is showed.</li>
```
Comment (Honry) [inline nwjs-iot/nwjsapi/chrome-extension-api/chrome-tabs/index.html]: As I know the there's no tab in nw.js app.

File: nwjs-iot/nwjsapi/tray/tray-event-click/index.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+</ol>
+<script>
+
+function out(id, msg) {
+  var h3 = document.createElement("h3");
+  h3.setAttribute("id", id);
+  h3.innerHTML = msg;
+  document.body.appendChild(h3);
+}
+var tray = new nw.Tray({title: "Tray", icon: "img/intel.png", tooltip: "click me", alticon: "g.png", iconsAreTemplates: true});
```
Comment (Honry) [inline nwjs-iot/nwjsapi/tray/tray-event-click/index.html]: Why use this kind of picture, g.png? Where does it come from?

File: nwjs-iot/nwjsapi/tray/tray-remove/index.html [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+</p>
+<ol>
+  <li>Click "Get Tray Parameters" to get tray parameters.</li>
+  <li>Click "Tray Remove" to remove tray, then click "Get Tray Parameters".</li>
+</ol>
+<p>
+  <strong>Expected Output:</strong>
+</p>
+<ol>
+  <li>Tray parameters are displayed.</li>
```
Comment (Honry) [inline nwjs-iot/nwjsapi/tray/tray-remove/index.html]: Should list the details parameters which should be displayed.

File: nwjs-iot/nwjsapi/webview/webview-enable-nodejs/index.html [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+</style>
+<p>
+  <strong>Test Steps:</strong>
+</p>
+<ol>
+  <li>Check if process version is obtained and "success" is showed.</li>
```
Comment (Honry) [inline nwjs-iot/nwjsapi/webview/webview-enable-nodejs/index.html]: If there's no operation need tester to do, just a <p> label with text like:
"Test passes if "Process.version is:v7.9.0" is showed and "success" is showed."

File: nwjs-iot/nwjsapi/webview/webview-load-localfile/index.html [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+  display: block;
+  height: 40%;
+  width: 100%;
+}
+</style>
+<p>
+  <strong>Test Steps:</strong>
+</p>
+<ol>
+  <li>Check if second window is loaded and text "success" is showed.</li>
```
Comment (Honry) [inline nwjs-iot/nwjsapi/webview/webview-load-localfile/index.html]: Ditto.

File: nwjs-iot/nwjsapi/chrome-extension-api/chrome-cookies/index.html (hunk 1 above)
Comment (xiaoyu3x) [inline nwjs-iot/nwjsapi/chrome-extension-api/chrome-cookies/index.html]: You are right!
I just modify expected result:
`<li>"Set Cookie Success" is showed and cookie value is "Hello Baidu".</li>`

File: nwjs-iot/nwjsapi/chrome-extension-api/chrome-proxy/index.html (hunk 2 above)
Comment (xiaoyu3x) [inline nwjs-iot/nwjsapi/chrome-extension-api/chrome-proxy/index.html]: This code has been update:
```
<p>
  <strong>Test Steps:</strong>
</p>
<ol>
  <li>Click "Set Proxy" button to set proxy "http:1.2.3.4:1080".</li>
  <li>Click "Get Proxy" button to get proxy.</li>
  <li>Click "Clear Proxy", then click "Get Proxy".</li>
</ol>
<p>
  <strong>Expected Output:</strong>
</p>
<ol>
  <li>Proxy is set and "success" is showed.</li>
  <li>Proxy is showed, for example: "host:1.2.3.4" "port":1080.</li>
  <li>Proxy is cleared and proxy couldn't be showed.</li>
</ol>

```

File: nwjs-iot/nwjsapi/chrome-extension-api/chrome-runtime-getPlatformInfo/index.html (hunk 3 above)
Comment (xiaoyu3x) [inline nwjs-iot/nwjsapi/chrome-extension-api/chrome-runtime-getPlatformInfo/index.html]: Expected output has been updated:
`<li>The platform info is shown,"os is: linux" and "arch is: x86-64" are showed if platform is linux x86-64.</li>`

File: nwjs-iot/nwjsapi/chrome-extension-api/chrome-tabs/index.html (hunk 4 above)
Comment (xiaoyu3x) [inline nwjs-iot/nwjsapi/chrome-extension-api/chrome-tabs/index.html]: No,
Nw.js support chrome api, this function belong to chrome api.

File: nwjs-iot/nwjsapi/tray/tray-event-click/index.html (hunk 5 above)
Comment (xiaoyu3x) [inline nwjs-iot/nwjsapi/tray/tray-event-click/index.html]: OK, code has been updated.

Comment (Honry) [inline nwjs-iot/nwjsapi/tray/tray-event-click/index.html]: I am asking you question....
please answer me.

Comment (xiaoyu3x) [inline nwjs-iot/nwjsapi/tray/tray-event-click/index.html]: OK, I know.
This g.png is found in upstream case, and I has used another intel picture to replace it.

File: nwjs-iot/nwjsapi/webview/webview-enable-nodejs/index.html (hunk 7 above)
Comment (Honry) [inline nwjs-iot/nwjsapi/webview/webview-enable-nodejs/index.html]: I meant no need "Test Steps" and "Expected Result", only a <p> label with above sentence.

**Comment by Honry** [comment]
LGTM

**Pull request merged**